packages/web3-providers/src/treasure/constants.ts

```diff
@@ -0,0 +1,44 @@
+import { NetworkType } from '@masknet/web3-shared-evm'
+
+export const TREASURE_ARBITRUM_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/treasureproject/marketplace'
+export const TREASURE_BRIDGEWORLD_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/treasureproject/bridgeworld'
+export const TREASURE_METADATA_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/treasureproject/metadata'
+export const TREASURE_SMOLVERSE_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/treasureproject/smolverse'
+export const TREASURE_MARKETPLACE_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/treasureproject/marketplace'
+export const TREASURE_REALM_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/jona/realm'
+
+export const METADATA_SUPPORT = [
```

Comment:
**NO BARE ADDRESSES.** Move them to `@masknet/web3-constants`. Please leave a referrer link.

Reply:
ref: https://github.com/TreasureProject/treasure-marketplace/blob/main/.env.production